roles/openshift_master/tasks/main.yml

```diff
@@ -24,10 +24,10 @@
   firewalld: port=4001/tcp permanent=true state=enabled
 
 - name: Open firewalld port for OpenShift
```

Comment:
You could also factorise the declaration of the firewalld in a more concise syntax like this:
```
- name: Open firewalld port for etcd embedded in OpenShift
  firewalld: port={{ item[0] }} permanent={{ item[1] }} state=enabled
  with_nested:
    - [ 4001/tcp, 8080/tcp, 8443/tcp, 8444/tcp ]
    - [ true, false ]
```
Port 8444 is used for web console